x/deployment/types/types.go

```diff
@@ -67,6 +61,16 @@ var GroupStateMap = map[string]GroupState{
 	"closed":             GroupClosed,
 }
 
+// GroupStateToReadable returns the adverb of the state
+func GroupStateToReadable(g GroupState) string {
```

Comment:
this shouldn't be needed anymore - we have the [stringer](https://github.com/ovrclk/akash/blob/8d399ba1e12bd6ae3be55718b6ebb2d91ccce71a/x/deployment/types/autogen_stringer.go#L41) for it.

Reply:
good point, cleans up the tests too